src/commons/services/utils.js

```diff
@@ -116,19 +116,20 @@
 
         vm.getClusterDetails = function(clusterId) {
             clusterName = "Unassigned";
-            if($rootScope.clusterData !== null && clusterId !== "") {
+
+            if($rootScope.clusterData !== null && typeof clusterId !== "undefined") {
```

Comment:
@kamleshverma1 
1) typeof operator helps us to identify whether the variable is declared or not along with whether the value is undefined or not.
2) The typeof operator is guaranteed to return a string. Direct comparisons against undefined are troublesome as undefined can be overwritten.